Frameworks/include/CALayerInternal.h

```diff
@@ -29,28 +29,24 @@ class CADisplayProperties {
     BOOL hidden;
     CGRect bounds;
     CGRect contentsRect, contentsCenter;
-    CGRect contentsInset; //  Texture insets
     uint32_t contentsOrientation;
```

Comment:
added a comment

---
In reply to: [86869414](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r86869414) [](ancestors = 86869414)